grail/steps.py

```diff
@@ -142,7 +142,7 @@ def print_to_console():
 
 
 def step(func=None, description='', pending=False, step_group=False, format_description=False,
-         treat_nested_steps_as_methods=False, log_output=True, log_input=True):
+         treat_nested_steps_as_methods=False, log_output=True, log_input=True, clean_params=True):
```

Comment:
It might be good to create a global setting to switch between these two ways. It'll help not to set this param on each method call.